common/log/zap_logger_bench_test.go

```diff
@@ -45,7 +45,7 @@ BenchmarkLoggerWithoutFields-4            162109              7211 ns/op
 */
 
 func BenchmarkZapLoggerWithFields(b *testing.B) {
-	zLogger := buildZapLogger(Config{Level: "info"}, false)
+	zLogger, _ := buildZapLogger(Config{Level: "info"}, false)
```

Comment:
Please check these errors and fail the benchmarks if these fail. It will help prevent problems when this code changes in the future